Sweetener.Logging/LogLevel.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Sweetener.Logging
+{
+    /// <summary>
+    /// Specifies a priority level and semantics associated with a log entry.
+    /// </summary>
+    public enum LogLevel
```

Comment:
I was reading that Trace is being phased out by the API and that Debug is preferred. Is that what you see too? I don’t really know what the difference is haha

Reply:
So I looked more into this. It looks like `Trace` is used for recording the progression of an application:

Eg.
```csharp
if (<condition>)
{
    logger.Trace("<Predicate> is true. Entered If");
    // ...
}

// ...

for (int i = 0; i < arr.Length; i++)
{
    logger.Trace("Loop: i = {0}", i);
    // ...
}
```

Some projects are reluctant to include `Trace` due to how verbose it can be. In other words, `Trace` encourages frequently logging, even in tight loops, which may add latency even when disabled. However, I think projects include it due to demand. Eg. SLF4J has this in their [FAQ](https://www.slf4j.org/faq.html#trace). That being said, I think it is almost always there.

Reply:
A developer that will abuse Trace can just as easily abuse Debug. But I guess that also means a developer that needs to use Trace can also use Debug. If it's being dropped by other major libraries then for consistency it makes sense to follow their example.